script.js

```diff
@@ -0,0 +1,55 @@
+var playerMove= "x"
+var nboard=9
+function onclick(playerMove);
```

Comment:
If you're *defining* a function, it needs curly braces:
```
function onclick(playerMove){
//  doSomething
}
```
If you're *calling* a function, you don't have the curly braces.